File: java/src/Algorithms/Two_Sum.java
package Algorithms;

/**
 * Created by gaojunliang on 9/18/15.
 */
public class Two_Sum {
    public static int[] twoSum(int[] nums, int target){
        int[] z={0,0};
        for(int i=0;i<nums.length-1;i++){
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target){
                    int[] a = {i+1,j+1};
                    return a;
                }
            }
        }
        return z;
    }

    public static void main(String[] args) {
        int[] numbers={2, 7, 11, 15};
        int target=9;
        System.out.println(twoSum(numbers,target)[0]+" "+twoSum(numbers,target)[1]);
    }
}

File: python/two-sum.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i in range(0, len(nums)-1):
            for j in range(i+1, len(nums)):
                if nums[i] + nums[j] == target: return [i, j]

def main():
    nums = [2, 7, 11, 15]
    target = 9
    s = Solution()
    r = s.twoSum(nums, target)
    print(r)

if __name__ == '__main__':
    main()
